src/net.cpp

```diff
@@ -1502,6 +1503,57 @@ void CConnman::ThreadAddressSeed()
     else{
         LogPrintf("DNS seeding disabled\n");
     }
+    // Fixed seeds
+    bool add_fixed_seeds = gArgs.GetBoolArg("-fixedseeds", DEFAULT_FIXEDSEEDS);
+
+    if (!add_fixed_seeds) {
```

Comment:
Can you just inline `GetBoolArg` here? You do update it later, but i think the updated version is not used at all?

Reply:
Good catch! It was needed in `ThreadOpenConnections`, but no longer after the move.